src/awkward/_connect/cling.py

```diff
@@ -148,7 +148,7 @@ class ArrayView {
     ssize_t start_;
     ssize_t stop_;
     ssize_t which_;
-    ssize_t* ptrs_;
+    ssize_t* ptrs_;    //! data pointer
     PyObject* lookup_; //! lookup
```

Comment:
I was trying to find out the reason of the [SystemError](https://github.com/root-project/root/issues/11602). Running the code in `lldb` showed the following error that is fixed by adding the comment: 
`Error in <TStreamerInfo::Build>: awkward::ArrayView, discarding: long* ptrs_, no [dimension]`

Reply:
So just all of the class's pointers? Is there any particular meaning to "`data pointer`" versus "`lookup`", or can any word go there?

Maybe it would be best to include a comment explaining what these `//!` things do; a link to the issue you pointed to above would be a step, at least. We don't want a future maintainer to look through this and say, "I don't know what this comment means, so it's not useful as a comment" and remove it.

Reply:
@agoose77 found this link:

https://github.com/root-project/root/blob/master/io/doc/TFile/README.md#streamerinfo

(https://github.com/scikit-hep/awkward/pull/1781#issuecomment-1282628108).